tree/treeplayer/src/TDFUtils.cxx

```diff
@@ -36,6 +36,79 @@ TIgnoreErrorLevelRAII::~TIgnoreErrorLevelRAII()
    gErrorIgnoreLevel = fCurIgnoreErrorLevel;
 }
 
+const std::type_info &TypeName2TypeID(const std::string &name)
```

Comment:
maybe we should leave a comment for posterity explaining that we do not deal with pointer/reference types because TDF does not accept columns of pointer type and references are removed before getting here